.github/workflows/check.yaml

```diff
@@ -1,7 +1,7 @@
 name: checks
 on: [push, pull_request]
 env:
-  GO_VERSION: 1.16
+  GO_VERSION: 1.17
```

Comment:
Can you use this PR to integrate with https://github.com/marketplace/actions/go-version-action ?

Reply:
Nice! I updated the config, let's see if it works